From 787c263605e876f36af23d27981819dd4087575c Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Tue, 31 Dec 2019 18:39:42 +0100
Subject: [PATCH 1/1] Solus tweaks to default kicker/kickoff

- Add shortcuts for Software Center in menu
- Change user icon to solus
---
 applets/kicker/package/contents/config/main.xml  | 2 +-
 applets/kickoff/package/contents/config/main.xml | 4 ++--
 applets/kickoff/package/contents/ui/Header.qml   | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/applets/kicker/package/contents/config/main.xml b/applets/kicker/package/contents/config/main.xml
index cddcad4..eb20d16 100644
--- a/applets/kicker/package/contents/config/main.xml
+++ b/applets/kicker/package/contents/config/main.xml
@@ -38,7 +38,7 @@
     </entry>
     <entry name="favoriteApps" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoriteSystemActions" type="StringList">
       <label>List of system action favorites.</label>
diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index be031dc..fba96eb 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -20,7 +20,7 @@
     </entry>
     <entry name="favorites" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover.desktop</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoritesPortedToKAstats" type="Bool">
       <label>Are the favorites ported to use KActivitiesStats to allow per-activity favorites</label>
@@ -28,7 +28,7 @@
     </entry>
     <entry name="systemApplications" type="StringList">
       <label>List of applications at the top of the "Computer" tab.</label>
-      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,org.kde.discover.desktop</default>
+      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="useExtraRunners" type="Bool">
       <label>Whether to use additional KRunner plugins to produce results in the search.</label>
diff --git a/applets/kickoff/package/contents/ui/Header.qml b/applets/kickoff/package/contents/ui/Header.qml
index c3d4d6a..0770cfa 100644
--- a/applets/kickoff/package/contents/ui/Header.qml
+++ b/applets/kickoff/package/contents/ui/Header.qml
@@ -86,7 +86,7 @@ Item {
     }
 
     PlasmaCore.IconItem {
-        source: "user-identity"
+        source: "solus-identity"
         visible: faceIcon.status !== Image.Ready
         width: units.gridUnit * 3
         height: width
-- 
2.24.1

